indico/modules/rb_new/client/js/common/roomSearch/reducers.js

```diff
@@ -67,7 +67,7 @@ export function initialRoomFilterStateFactory(namespace) {
         });
     }
 
-    return state;
+    return Object.assign(state, customFilters);
```

Comment:
we already conditionally add default state above based on the namespace. what about doing the same for calendar as well instead of passing the customFilters?